lib/android-helpers.js

```diff
@@ -175,8 +176,8 @@ helpers.getDeviceInfoFromCaps = async function (opts = {}) {
 };
 
 // returns a new adb instance with deviceId set
-helpers.createADB = async function (javaVersion, udid, emPort, adbPort, suppressKillServer) {
-  let adb = await ADB.createADB({javaVersion, adbPort, suppressKillServer});
+helpers.createADB = async function (javaVersion, udid, emPort, remoteAdbHost, adbPort, suppressKillServer) {
+  let adb = await ADB.createADB({javaVersion, remoteAdbHost, adbPort, suppressKillServer});
```

Comment:
Can we just move `remoteAdbHost` after `suppressKillServer` to make it a non-breaking change?